sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>Did you use the variable <code>original_version</code>  later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2043/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Yes. I use `original_version` in the final cell of the notebook. 
It's this line: `%pip install 'sagemaker=={}'.format(original_version)`